File: app/src/main/java/com/test/imageload/SplashActivity.java
package com.test.imageload;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;

import com.test.imageload.kotlin.KotlinActivity;
import com.test.imageload.lifecycle.LifeCycleActivity;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class SplashActivity extends Activity {

    @BindView(R.id.textView)
    TextView textView;
    private Activity act;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        act = this;
        setContentView(R.layout.activity_splash);
        ButterKnife.bind(this);
    }

    @OnClick(R.id.textView)
    public void onViewClicked() {

        Intent intent = new Intent(act, MainActivity.class);
        startActivity(intent);
    }

    @OnClick(R.id.splash)
    public void onSplashViewClicked() {
        Intent intent = new Intent(act, KotlinActivity.class);
        startActivity(intent);
    }

    @OnClick(R.id.tv_glide)
    public void onGlideViewClicked() {
        Intent intent = new Intent(act, LifeCycleActivity.class);
        startActivity(intent);
    }
}

File: app/src/main/java/com/test/imageload/SplashActivity.kt
package com.test.imageload

import android.content.Intent
import android.os.Bundle
import butterknife.ButterKnife
import butterknife.OnClick
import com.test.imageload.base.BaseActivity
import com.test.imageload.db.BaseDaoFactory
import com.test.imageload.db.DbTest
import com.test.imageload.db.UserInfo
import com.test.imageload.db.UserInfoDao
import com.test.imageload.kotlin.MainActivity
import com.test.imageload.picture.lifecycle.LifeCycleActivity

class SplashActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        ButterKnife.bind(this)

    }

    override fun getLayoutId(): Int {
        return R.layout.activity_splash
    }

    @OnClick(R.id.splash)
    fun onSplashViewClicked() {
        val intent = Intent(act, MainActivity::class.java)
        startActivity(intent)
    }

    @OnClick(R.id.tv_glide)
    fun onGlideViewClicked() {
        val intent = Intent(act, LifeCycleActivity::class.java)
        startActivity(intent)
    }
}
